load_testing/lib/flow_ial2_proofing.py

```diff
@@ -14,28 +17,30 @@ def do_ial2_proofing(context):
     * license_back - Image data for back of driver's license
     """
     # Request IAL2 Verification
-    resp = do_request(context, "get", "/verify", "/verify/doc_auth")
+    resp = do_request(context, "get", "/verify", "/verify/welcome")
     auth_token = authenticity_token(resp)
 
     # Post consent to Welcome
     resp = do_request(
         context,
         "put",
-        "/verify/doc_auth/welcome",
-        "/verify/doc_auth/agreement",
+        "/verify/welcome",
+        "/verify/agreement",
         "",
         {"authenticity_token": auth_token, },
     )
     auth_token = authenticity_token(resp)
 
+    if os.getenv("DEBUG"):
+        print("DEBUG: /verify/agreement")
```

Comment:
can we use the `logging.debug` from the other files here?

Reply:
Yeah, I created an issue to do that here https://github.com/18F/identity-loadtest/issues/85. This PR already had enough scope creep.